lib/ex_doc/retriever.ex

```diff
@@ -267,14 +269,14 @@ defmodule ExDoc.Retriever do
     end
   end
 
-  defp get_callbacks(:behaviour, module, source_path, source_url, abst_code) do
+  defp get_callbacks(:behaviour, module, config, abst_code) do
     optional_callbacks = get_optional_callbacks(abst_code)
     (Code.get_docs(module, :callback_docs) || [])
     |> Enum.sort_by(&elem(&1, 0))
-    |> Enum.map(&get_callback(&1, source_path, source_url, optional_callbacks, abst_code))
+    |> Enum.map(&get_callback(&1, config.source_path, config.source_url, optional_callbacks, abst_code))
```

Comment:
If we store the module name and type, we will also be able to pass the `module_info` to other functions such as `get_types`. :)